ironfish/src/memPool/memPool.ts

```diff
@@ -47,10 +48,12 @@ export class MemPool {
 
     this.queue = new PriorityQueue<MempoolEntry>(
       (firstTransaction, secondTransaction) => {
-        if (firstTransaction.fee === secondTransaction.fee) {
+        const firstTransactionFeeRate = this.getFeeRate(firstTransaction)
+        const secondTransactionFeeRate = this.getFeeRate(secondTransaction)
+        if (firstTransactionFeeRate === secondTransactionFeeRate) {
           return firstTransaction.hash.compare(secondTransaction.hash) > 0
         }
-        return firstTransaction.fee > secondTransaction.fee
+        return firstTransactionFeeRate > secondTransactionFeeRate
```

Comment:
It's miner but it would be nice if this was consistent with https://github.com/iron-fish/ironfish/blob/master/ironfish/src/primitives/blockheader.ts#L28-L42